sensors/simulated/sensorhub-driver-fakecam/build.gradle

```diff
@@ -18,6 +18,12 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.fakecam.Activator')
+    attributes('Import-Package': '''
+      !android.util.*, !java.*, !java.*, !javax.crypto.*, !javax.imageio.*, !javax.net.*, !javax.naming.*, 
+      !javax.management.*, !javax.security.*, !javax.sql.*, !javax.swing.*, !javax.transaction.*, !javax.xml.*,
+      !org.w3c.dom.*, !org.xml.sax.*,!com.sun.*,
+      org.osgi.service.repository;resolution:=optional, 
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
No need, Ok for building and installing.: 99c19942d89674dfccad83baa8b12f2b9ee1ef46